interface/resources/controllers/keyboardMouse.json

```diff
@@ -72,156 +72,86 @@
-                ["Keyboard.A"],
-                ["Keyboard.D"]
-            ]
-          },
-          "when": ["Application.CameraThirdPerson", "!Keyboard.Control"],
-          "to": "Actions.Yaw"
-        },
-        
-        { "from": { "makeAxis" : [
-                ["Keyboard.A"],
-                ["Keyboard.D"]
-             ]
-          },
-          "when": ["Application.CameraLookAt", "!Keyboard.Control"],
-          "to": "Actions.Yaw"
+        { "from": "Keyboard.D",
+          "when": ["!Application.CameraFSM", "!Application.CameraIndependent", "!Application.CameraEntity", "Application.CaptureMouse", "!Keyboard.Control"],
+          "to": "Actions.LATERAL_RIGHT"
         },
 
-        { "from": { "makeAxis" : [
-                ["Keyboard.A"],
-                ["Keyboard.D"]
-             ]
-          },
-          "when": ["Application.CameraSelfie", "!Keyboard.Control"],
-          "to": "Actions.Yaw"
-        },
-        
         { "from": { "makeAxis" : [
                 ["Keyboard.TouchpadLeft"],
                 ["Keyboard.TouchpadRight"]
             ]
           },
-          "when": "Application.CameraFirstPerson",
+          "when": ["!Application.CameraFSM", "!Application.CameraIndependent", "!Application.CameraEntity"],
           "to": "Actions.Yaw"
         },
 
-        { "from": { "makeAxis" : [
-                ["Keyboard.TouchpadLeft"],
-                ["Keyboard.TouchpadRight"]
-            ]
-          },
-          "when": "Application.CameraFirstPersonLookat",
-          "to": "Actions.Yaw"
-        },
-        
-        { "from": { "makeAxis" : [
-                ["Keyboard.TouchpadLeft"],
-                ["Keyboard.TouchpadRight"]
-            ]
-          },
-          "when": "Application.CameraThirdPerson",
-          "to": "Actions.Yaw"
-        },
-        
-        { "from": { "makeAxis" : [
-                ["Keyboard.TouchpadLeft"],
-                ["Keyboard.TouchpadRight"]
-            ]
-          },
-          "when": "Application.CameraLookAt",
-          "to": "Actions.Yaw"
-        },
-        
-        { "from": { "makeAxis" : [
-                ["Keyboard.TouchpadLeft"],
-                ["Keyboard.TouchpadRight"]
+        { "from": { "makeAxis" : ["Keyboard.MouseMoveLeft", "Keyboard.MouseMoveRight"] },
+          "when": ["Keyboard.RightMouseButton", "!Application.CaptureMouse"],
+          "to": "Actions.DeltaYaw",
+          "filters":
+            [
+                { "type": "scale", "scale": 0.6 }
             ]
-          },
-          "when": "Application.CameraSelfie",
-          "to": "Actions.Yaw"
         },
 
         { "from": { "makeAxis" : ["Keyboard.MouseMoveLeft", "Keyboard.MouseMoveRight"] },
-          "when": "Keyboard.RightMouseButton",
           "to": "Actions.DeltaYaw",
+          "when": "Application.CaptureMouse",
           "filters":
             [
-                { "type": "scale", "scale": 0.6 }
+                { "type": "scale", "scale": 0.2 }
```

Comment:
Is this change of scale value intentional? Does it adversely affect current sensitivity / speed of movement?

Reply:
This is a matter of the diff being confusing.  The block above this handles the old cases when !Application.CaptureMouse, and still has scale 0.6.  because of how the mouse capture mode works, we want this to be less sensitive.  then on top of that, you can now control the sensitivity further with Camera.sensitivity (with the caveat of framerate-dependent behavior as discussed previously)